src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -846,7 +845,7 @@ const WorkflowView = _.flow(
                     div({ style: { display: 'inline-block', marginLeft: '0.25rem', width: sourceRepo === 'agora' ? 75 : 200 } }, [
                       h(Select, {
                         id,
-                        isDisabled: !!WorkspaceUtils.editWorkspaceError(ws),
+                        isDisabled: !WorkspaceUtils.canEditWorkspace.value,
```

Comment:
shoudn't this be .canEditWorkspace(ws).value ?

Reply:
Yikes, thank you. It's a little scary that these can leak through automation.

Reply:
Yeah, some of the areas that haven't been actively developed recently have next to no test coverage.